### PR TITLE
Trim long URLs on admin dash.

### PR DESCRIPTION
**Pull request by overint**
As per #258, implemented URL trimming. URL is shown on hover, and when clicked is opened in a new tab.
I also made the URL columns wider, since they were unnecessarily small.

Screenshots:
http://puu.sh/sUwWV/1577eed220.png
http://puu.sh/sUwXw/0b6fbcd696.png

### CODE REVIEW

File: public/js/AdminCtrl.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ polr.controller('AdminCtrl', function($scope, $compile) {
                 "ajax": BASE_API_PATH + 'admin/get_admin_links',
 
                 "columns": [
-                    {className: 'wrap-text', data: 'short_url', name: 'short_url'},
-                    {className: 'wrap-text', data: 'long_url', name: 'long_url'},
+                    {data: 'short_url', name: 'short_url'},
+                    {data: 'long_url', name: 'long_url', width: '30%'},
```
Comment (cydrobolt) [inline public/js/AdminCtrl.js]: What are the percentages for?

Comment (overint) [inline public/js/AdminCtrl.js]: I tested both the previous tables (pre this change) and this pull, both of them seem to function poorly on mobile. 
[Screenshot from polr demo admin](http://puu.sh/sVaj1/401023a6bf.png)

Comment (cydrobolt) [inline public/js/AdminCtrl.js]: Ah, I see. Could we remove this change for now? We can handle the width using DataTables' auto width adjustment setting.

Comment (overint) [inline public/js/AdminCtrl.js]: I can remove it, however I did try doing it with data tables and I did not have any success in getting it working correctly.

File: public/js/AdminCtrl.js [hunk 2]
```diff
@@ -45,8 +45,8 @@ polr.controller('AdminCtrl', function($scope, $compile) {
                 "ajax": BASE_API_PATH + 'admin/get_admin_links',
 
                 "columns": [
-                    {className: 'wrap-text', data: 'short_url', name: 'short_url'},
-                    {className: 'wrap-text', data: 'long_url', name: 'long_url'},
+                    {data: 'short_url', name: 'short_url'},
```
Comment (cydrobolt) [inline public/js/AdminCtrl.js]: Shouldn't we still wrap-text even if we're trimming the link? Otherwise, the link will bleed into the neighboring fields.

Comment (overint) [inline public/js/AdminCtrl.js]: Fixed.

**Comment by cydrobolt** [comment]
Thank you again for your contributions! :smile:

**Comment by overint** [comment]
No worries!

**Pull request merged**